Meme-ify-Me/app/src/main/java/nyc/c4q/rosmaryfc/meme_ify_me/VanillaMemeEdit.java

```diff
@@ -0,0 +1,39 @@
+package nyc.c4q.rosmaryfc.meme_ify_me;
+
+import android.support.v7.app.ActionBarActivity;
+import android.os.Bundle;
+import android.view.Menu;
+import android.view.MenuItem;
+
+
+public class VanillaMemeEdit extends ActionBarActivity {
```

Comment:
I like the logic here of starting an individual intent for whichever kind of meme the user has selected